src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -163,10 +163,6 @@ function toggleCollapse(e, pans) {
         var i = 0;
         for (var idx = 0; idx < panels.length; idx++) {
             if ($(panels[idx]).attr('class').indexOf('in') === -1) {
-
-                // The timeout value '50' is being used in InstructorFeedbackResultsPage.verifyAllResultsPanelBodyVisibility()
-                // and InstructorFeedbackResultsPageUiTest.testPanelsCollapseExpand()
-                // Therefore, when changing this timeout value, please update the waiting times accordingly
```

Comment:
Seems to have been removed as the comments do not reflect the current implementation state